src/subgame.h

```diff
@@ -51,10 +51,7 @@ struct SubgameSpec
 		menuicon_path(menuicon_path_)
 	{}
 
-	bool isValid() const
-	{
-		return (id != "" && path != "");
-	}
+	bool isValid() const { return !id.empty() && !path.empty(); }
```

Comment:
Why is this now on one line?

Also, !(id.empty() || path.empty()) might be clearer